spec/Task_Parallelism_and_Synchronization.tex

```diff
@@ -836,6 +836,10 @@ \section{Task Intents}
   formal-intent identifier, task-intent-list
 \end{syntax}
 
+where the following intents can be used as a \sntx{formal-intent}:
+the blank intent, \chpl{ref}, \chpl{const ref}, \chpl{const},
+\chpl{const in}, \chpl{in}.
```

Comment:
What's the rationale for the order of the non-blank intents above?  (my best guess: try to maximize the degree to which similar things are adjacent to one another).  It seems odd to me to put the intentionally vague 'const' between the two more precise equivalents.  I'd probably order them:  'ref', 'in', 'const', 'const in', 'const ref' for readability and to sort the consts together.  Or, alphabetically.

This doesn't need to happen today, but I think that, ultimately, we should introduce a task-formal-intent syntactic rule that expands to the legal intents rather than qualifying which intents are legal in the text of this paragraph would be far preferable (assuming we never intent to support 'out' and 'inout' which I think remains up for debate and would arguably be a reason to leave things as-is).

(Editorializing: Personally, I continue to dislike permitting 'blank' as a task intent because I don't see the value over simply not listing that identifier and it seems more likely to be confusing/indicate confusion than helpful).